content/intermediate/201_resource_management/basic-pod-limits.md

```diff
@@ -1,22 +1,22 @@
 ---
 title: "Basic Pod CPU and Memory Management"
-date: 2020-06-22T00:00:00-03:00
+date: 2021-11-10T00:00:00-03:00
 weight: 10
 draft: false
 ---
 
 We will create four pods:
```

Comment:
This feels out of place with the **Deploy Metrics Server** section. Consider putting metrics server on it's own page as a module prerequisite, or at least moving above deploying the pods.